WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Helpers/BlogDashboardState.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+class BlogDashboardState {
+    static let shared = BlogDashboardState()
+
+    /// If the dashboard has ever loaded before
+    var hasEverLoaded = false
+
+    /// If loading the cards in the dashboard failed
+    var loadingFailed = false
+
+    /// If the dashboard is currently being loaded for the very first time
+    /// aka: it has never been loaded before.
+    var isCurrentlyLoadingForTheVeryFirstTime: Bool {
```

Comment:
Feel free to ignore this suggestion. Maybe a more concise name?
```suggestion
    var isFirstLoad: Bool {
```

Reply:
Done in 00fea85e7e4789833e9d3be3a378d420e8d2e974